test/test_site/testPageVariables.md

```diff
@@ -0,0 +1,7 @@
+<variable name="included_variable_overriding_page_variable">Page Variable Overridden by Included Variable</variable>
+
+# Page Variables Should Not Leak Into Inner Files
+Should be blank: {{ page_variable }}
```

Comment:
The current behavior seems to have no difference with `{% set ...=... %}` (the only additional benefit we get is that you can declare it in any part of the page, but that's it :/).

Is it not possible to bring page variable into included files?

Reply:
Also the name of this file is somewhat misleading; it tests the page variables w.r.t. it being used inside an `<include>`, not as a _general_ page for testing page variables.

Reply:
Ahh yes bringing them in is not a problem, but I was wondering if we should actually in the original post:

>Should variables cascade into includes? I'm thinking no, it can be confusing if the user is trying to trace where a variable came from and can't find it. If the user wishes for them to be used they can include it specifically:
>
> ```html
> <variable name="x">5</variable>
> <include ... >
>   <span id="x">{{ x }}</span>
> </include>
> ```